doc/source/ray-observability/getting-started.rst

```diff
@@ -235,9 +235,9 @@ Use the Actors view to see the logs for an Actor and which Job created the Actor
     <div style="position: relative; height: 0; overflow: hidden; max-width: 100%; height: auto;">
         <iframe width="560" height="315" src="https://www.youtube.com/embed/MChn6O1ecEQ" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
     </div>
-    
+
 The information for up to 1000 dead Actors is stored.
```

Comment:
The default value for `RAY_maximum_gcs_destroyed_actor_cached_count` is `100000`